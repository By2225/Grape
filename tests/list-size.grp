fun Int main() {
    List<Edge<String> > a;
    Edge<String> b = <<-"hi"->>;
    a = [b];
    print(size(a));
    return 0;
}
