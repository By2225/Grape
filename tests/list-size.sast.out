
fun Int main() {
List<Edge<String->*>> a;
Edge<String->*> (Edge<String->*> : b = (Edge<String->*> : -(String : hi)-));
(List<Edge<String->*>> : a = (List<Edge<String->*>> : [(Edge<String->*> : b)]));
(* : print((Int : size((List<Edge<String->*>> : a)))));
return (Int : 0);
}
